toast/ios/Plugin/ToastLabel.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+@objc public class ToastLabel: UILabel {
+    private struct AssociatedKeys {
+        static var padding = UIEdgeInsets()
+    }
+
+    public var padding: UIEdgeInsets? {
+        get {
+            return objc_getAssociatedObject(self, &AssociatedKeys.padding) as? UIEdgeInsets
+        }
+        set {
+            if let newValue = newValue {
+                objc_setAssociatedObject(self, &AssociatedKeys.padding, newValue as UIEdgeInsets, objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC)
+            }
+        }
+    }
```

Comment:
This originally used an associated object because extensions can't add properties. But since this is now a subclass, it can be much simpler. 
```suggestion
    var padding: UIEdgeInsets?
```